src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to CSS should have a `.css` extension, not `.scss`. SCSS files are preprocessed and should be compiled to CSS before being linked in HTML.